### PR TITLE
Test and fix issue #90

### PR DESCRIPTION
**Pull request by dafortin**
#patch#

Test fix issue #90

### CODE REVIEW

**Comment by ember-cli-visual-acceptance** [comment]
# Visual Acceptance tests:
## New

#### Firefox: Reproduce
 

#### Chrome: Reproduce
 

#### PhantomJS: Reproduce
 

#### SlimerJS: Reproduce

File: lib/commands/travis-visual-acceptance.js [hunk 1]
```diff
@@ -91,11 +100,14 @@ module.exports = {
     process.env.NO_ASSERT = true
 
     var prNumber = process.env.TRAVIS_PULL_REQUEST
+
     if (prNumber !== false && prNumber !== 'false' && process.env.VISUAL_ACCEPTANCE_TOKEN) {
       return runCommand(this, 'br').then(buildReport, function (params) {
```
Comment (EWhite613) [inline lib/commands/travis-visual-acceptance.js]: Need to add the if statement to the first param as well.
`.then(buildReport, ....)` ->
```
.then( function (params){
        if (options.msgOn === options.msgVar) {
           return buildReport(params)
        }
}, ....
```

Comment (dafortin) [inline lib/commands/travis-visual-acceptance.js]: Sorry not sure to understand what you mean here.

Comment (dafortin) [inline lib/commands/travis-visual-acceptance.js]: I was planning to wrap the full runCommand in the if

```
if (options.msgOn === options.msgVar) {
    return runCommand(this, 'br').then(buildReport, function (params) { ... })
```

Comment (EWhite613) [inline lib/commands/travis-visual-acceptance.js]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then

Currently ur only handling `onRejected` (`ember test` failed`). You need to handle `onPassed` with  
```
if (options.msgOn === options.msgVar) {
           return buildReport(params)
        }
```
as well

**Comment by sglanzer** [comment]
Approved

**Pull request merged**